client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -809,26 +793,24 @@ private void endLRA(URI lra, boolean confirm, String compensator, String userDat
                     .header(NARAYANA_LRA_PARTICIPANT_DATA_HEADER_NAME, userData)
                     .async()
                     .put(Entity.text(""))
-                    .get(END_TIMEOUT, TimeUnit.SECONDS);
+                    .get(END_TIMEOUT*100, TimeUnit.SECONDS);
```

Comment:
Please can you mention why we need "*100"?

Reply:
oh dear, I thought I'd removed those, well spotted - it was to give me more time to use the debugger since jax rs calls to the coordinator timeout before I've had a chance to debug on the coordinator side.